docs/mpe/simple.md

```diff
@@ -7,7 +7,7 @@ action-shape: "(5)"
 action-values: "Discrete(5)"
 observation-shape: "(4)"
 observation-values: "(-inf,inf)"
-import: "pettingzoo.mpe.simple_v0"
+import: "pettingzoo.mpe import simple_v1"
```

Comment:
Good find.

Just a little change: this should be `from pettingzoo.mpe import simple_v1`.

Same with all the other environments